invisible_cities/reco/corrections_new.py

```diff
@@ -229,11 +310,19 @@ def apply_all_correction_single_maps(map_e0     : ASectorMap,
         raise MissingArgumentError
         pass
 
+    if norm_strat is norm_strategy.max:
+        norm_value = amap_max(map_e0).e0
+    elif norm_strat is norm_strategy.mean:
+        norm_value = np.mean(np.mean(maps.e0))
+    elif norm_strat is norm_strategy.custom:
+        if norm_value is None:
+            raise ValueError
+    else:
+        raise ValueError
+
```

Comment:
The exceptions should contain more explanations;  as norm_value has to be float; and norm_strategy not recognized